sdk/src/client/node_api/core/routes.rs

```diff
@@ -376,19 +377,49 @@ impl ClientInner {
     pub async fn get_utxo_changes_by_slot(&self, slot_index: SlotIndex) -> Result<UtxoChangesResponse> {
         let path = &format!("api/core/v3/commitments/by-slot/{slot_index}/utxo-changes");
 
-        self.get_request(path, None, false, true).await
+        self.get_request(path, None, false).await
     }
 
     /// Get all full UTXO changes of a given slot by its index.
     /// GET /api/core/v3/commitments/by-slot/{slot}/utxo-changes/full
     pub async fn get_utxo_changes_full_by_slot(&self, slot_index: SlotIndex) -> Result<UtxoChangesFullResponse> {
         let path = &format!("api/core/v3/commitments/by-slot/{slot_index}/utxo-changes/full");
 
-        self.get_request(path, None, false, false).await
+        self.get_request(path, None, false).await
     }
 }
 
 impl Client {
+    /// GET /api/core/v3/info endpoint
+    pub(crate) async fn get_permanode_info(node: &Node) -> Result<PermanodeInfoResponse> {
+        let mut node = node.clone();
```

Comment:
bit weird to expect a `&Node` when we actually need a clone. Can't we not just expect the same arguments and create a `Node` struct?

Reply:
We need a clone in any case because of the url, but I changed it so this function accepts `Node`